src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -915,6 +915,39 @@ ApiGlobe.prototype.removeEventListenerLayerChanged = function removeEventListene
     this.viewerDiv.removeEventListener('layerchanged:index', this.callbackLayerChanged, false);
 };
 
+/**
+ * Get the Attribution of all layers in the scene.
+ * @return {map}  A Map of attribution.
+ */
+
+ApiGlobe.prototype.getLayersAttribution = function getLayersAttribution() {
+    const lc = this.scene.getMap().layersConfiguration;
+    const map = new Map();
+    [...lc.getColorLayers(), ...lc.getElevationLayers()].forEach((l) => {
+        if (l.options.attribution) {
+            map.set(l.options.attribution.name, l.options.attribution);
+        }
+    });
+    return map;
+};
+
+/**
+ * Set the attribution for a layer.
+ * @param {param} Param - The id of the layer and the new IPR {id, attribution}.
+ */
+
+ApiGlobe.prototype.setLayerAttribution = function setLayerAttribution(param) {
```

Comment:
I'm not sure this function is useful at all.
Is there a use case for modifying attribution at runtime ?
```js
api.setLayerAttribution({id: layer.id, attribution: { ...});
```
can be written in a simpler way:
```js
layer.attribution = { ... }
```
WDYT?